packages/components/src/progress_bar/utils.jsx

```diff
@@ -0,0 +1,9 @@
+export const getPercentage = (input) => {
```

Comment:
`getPercentageString` maybe? 

Also, tests would be good for this one, to illustrate the possible cases that can occur for this util.